src/firefly/js/charts/ui/PlotlyWrapper.jsx

```diff
@@ -199,7 +199,8 @@ export class PlotlyWrapper extends Component {
 
             if (!useScatterGL) {
                 // when using SVG, it's actually faster to redraw then to do multiple updates
-                if (renderType === RenderType.RESTYLE_AND_RELAYOUT || dataUpdate.length > 1) {
+                if (renderType === RenderType.RESTYLE_AND_RELAYOUT || dataUpdate.length > 1 ||
+                    (data.find((d) => get(d, 'type').includes('histogram2d')))) {
```

Comment:
we need a comment here to explain the workaround or issue with histogram 2d

Reply:
done